example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -0,0 +1,135 @@
+---
+
+- name: Restore tokens
+  hosts: all
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Stop Scylla service
+      service:
+        name: scylla-server
+        state: stopped
+
+    - name: Remove Scylla directories
+      file:
+        path: "/var/lib/scylla/{{ item }}"
+        state: absent
+      loop:
+        - data
+        - commitlog
+        - hints
+        - view_hints
+
+    - name: Recreate Scylla directories
```

Comment:
Is it really needed? AFAIK scylla is going to create them by itself which all corresponding permissions.
Better drop this.

Reply:
Unfortunately this does not work

```
Apr 27 12:25:54 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: /var/lib/scylla/data doesn't exist - skipping
Apr 27 12:25:54 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: /var/lib/scylla/commitlog doesn't exist - skipping
Apr 27 12:25:54 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: ERROR: 'disks' tuning was requested but no disks were found. Your system can't be tuned until the issue is fixed.
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: Traceback (most recent call last):
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: File "/opt/scylladb/scripts/libexec/scylla_prepare", line 67, in <module>
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: run("{} --options-file /etc/scylla.d/perftune.yaml".format(perftune_base_command()))
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: File "/opt/scylladb/scripts/scylla_util.py", line 329, in run
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: return subprocess.check_call(cmd, shell=shell, stdout=stdout, stderr=stderr, env=scylla_env)
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: File "/opt/scylladb/python3/lib64/python3.7/subprocess.py", line 363, in check_call
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: raise CalledProcessError(retcode, cmd)
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal scylla_prepare[15724]: subprocess.CalledProcessError: Command '['/opt/scylladb/scripts/perftune.py', '--tune', 'disks', '--dir', '/var/lib/scylla/data', '--dir', '/var/lib/scyll
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal systemd[1]: scylla-server.service: control process exited, code=exited status=1
Apr 27 12:25:55 ip-172-31-0-12.eu-central-1.compute.internal systemd[1]: Failed to start Scylla Server.
-
```

Reply:
Ugr... Right.  perftune.py runs before the initial invocation of scylla.